### ISSUE TITLE
Instant answers: Calculator, Unit-converter, etc.

### ISSUE THREAD

**Issue by KirillOsenkov** [spectresystems/jarvis#67]
I maintain a website with some instant answers: http://quickinfo.io. 
Here's what it can do: http://quickinfo.io/?%3F (or just type `demo` there)

I've recently refactored out the engine that generates answers and decoupled it from the Html backend so it can provide answers for console, mobile etc.

Would you consider adding instant answers similar to the above? On a Mac pressing Cmd+Space supports math calculator, etc. You can type `2+3` and get `5` instantly. I thought it'd be a great idea for Jarvis. Let me know what your thoughts are and I'd be happy to help if needed.

**Comment by thecodejunkie** [comment]
👍 on this one

**Comment by patriksvensson** [comment]
@KirillOsenkov That looks really cool! Good work!

Do you provide some kind of API for it, or is there a NuGet package that Jarvis can consume? It would be really nice if Jarvis could visualize results differently like `Quickinfo` does as well (tables, boxes etc). Will have to look into that.

**Comment by KirillOsenkov** [comment]
Yup, there's an API and a NuGet package. Just published some info here:
https://github.com/KirillOsenkov/QuickInfo/blob/master/README.md#api-and-architecture

Yes, you can visualize the results however you want, there's the Node data structure that you can visit to build whatever UI you like. See the HtmlRenderer as an example.

Also this is very hot off the press, so any feedback, suggestions or requests are appreciated! I'm also considering writing a Xamarin iOS app for iPhone using this.

**Comment by HTMLGuyLLC** [comment]
+1 and reminder bump
